alphalens/utils.py

```diff
@@ -639,7 +674,8 @@ def get_clean_factor_and_forward_returns(factor,
                                          groupby_labels=None,
                                          max_loss=0.35,
                                          zero_aware=False,
-                                         cumulative_returns=True):
+                                         cumulative_returns=True,
+                                         is_returns=False):
```

Comment:
Nice catch on the docstring. I'll add it in. The parameter is used here: https://github.com/quantopian/alphalens/pull/361/files#diff-5195c0fa99041f5ae59720a046520bf0R824. The idea is that the user should be able to provide *either* raw pricing data (which gets converted into forward returns) *or* they can provide forward returns directly. The added flexibility is helpful for the use case where someone is getting returns data from pipeline (instead of pricing data).

Reply:
(sorry for not having caught this earlier) Is `is_return` option really useful? `utils.get_clean_factor` is already available as the entry point when forward returns are available instead of pricing data, that function was made available exactly for this purpose some time ago. I feel like it is more confusing than helpful to have `is_return` option. What do you think?

Reply:
You're right, I think this argument and the subsequent branch in the method should be removed. Instead, the user should use `utils.get_clean_factor`. Thanks for pointing this out, guys!